README.md

```diff
@@ -9,25 +9,25 @@ Web-app to visualize nodes and links on a map for Freifunk open mesh network.
 _Some similar features might have been implemented/merged_
 
 - Map layer modes (Allow to set a default layer based on time combined with a stylesheet)
-- Updates for selected node or list (incl. image stats cache-breaker)
-- Node filter is implemented
+- Automatic updates for selected node or list (incl. image stats cache-breaker)
```

Comment:
makes it more clear, that not something was updated by the maintainer in the code, but automatic updates were introduced for the user